tests/publiccloud/run_ltp.pm

```diff
@@ -36,21 +36,34 @@ sub wait_for_guestregister
 
 sub run {
     my ($self) = @_;
-    my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/home:/metan/' . generate_version() . '/home:metan.repo');
+    my $ltp_repo = get_var('LTP_REPO', 'http://download.suse.de/ibs/QA:/Head/' . generate_version("-") . '/x86_64/');
+    my $REG_CODE = get_required_var('SCC_REGCODE');
     $self->select_serial_terminal;
 
+    my $out       = script_output("curl $ltp_repo | grep ltp | head -1");
```

Comment:
Only a hint:
This [Mojo::UserAgent](https://mojolicious.org/perldoc/Mojo/UserAgent) is also a nice way to retrieve and parse html files. We use it for instance [here](https://gitlab.suse.de/openqa/scripts/blob/master/trigger_public_cloud.pl#L36)

Reply:
ok, I will try to use that instead of ugly regex :)

Reply:
done